python/lsst/ts/standardscripts/base_take_image.py

```diff
@@ -153,6 +195,30 @@ def set_metadata(self, metadata):
             * nimages
         )
 
+    def set_additional_nextvisit_metadata(self, metadata):
+        metadata.nimages = len(self.config.exp_times)
+        metadata.coordinateSystem = MetadataCoordSys.ICRS
+        if (self.ra is not None) and (self.dec is not None):
+            radec_icrs = ICRS(
+                Angle(self.nextVisit_metadata["ra"], unit=astropy.units.hourangle),
```

Comment:
As I mentioned above, the metadata session is already part of `self.config`, so instead of duplicating this information just use it from it, e.g.:

```
Angle(self.config.nextVisit_metadata["ra"], unit=astropy.units.hourangle)
```

but also, please, rename this to either `next_visit_metadata` or `visit_metadata` or, preferably, just `metadata`.